packages/styled-components/package.json

```diff
@@ -63,20 +63,21 @@
     "babel-plugin-styled-components": ">= 1",
     "css-to-react-native": "^2.2.2",
     "prop-types": "^15.5.4",
-    "react-is": "^16.8.0",
     "stylis": "^3.5.0",
     "stylis-rule-sheet": "^0.0.10",
     "supports-color": "^5.5.0"
   },
   "peerDependencies": {
-    "react": ">= 16.8.0",
-    "react-dom": ">= 16.8.0"
+    "react": ">= 16.8",
+    "react-dom": ">= 16.8",
+    "react-is": ">= 16.8"
```

Comment:
This is helpful, I've had to use `resolutions` to update these dependencies in consuming packages. Thanks for changing it 👍